Make connection login and password TEXT

Sighery opened the pull request:
This commit will change the connection `login` and `password` fields to be of type `TEXT`, so it won't have a length limitation anymore.

Closes #32657

hussein-awala reviewed (commented):
It appears like MySQL doesn't like your change:
```
sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (1118, 'Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs')
```

uranusjr commented on airflow/migrations/versions/0130_2_8_0_make_connection_login_password_text.py:
```diff
@@ -0,0 +1,60 @@
+    """Apply Make connection login and password text"""
+    with op.batch_alter_table(TABLE_NAME) as batch_op:
+        batch_op.alter_column(
+            "login", existing_type=sa.VARCHAR(length=500), type_=sa.TEXT(), existing_nullable=True
```
These types should match the declared types in models.

Sighery replied:
You're right. They should be matching now. Could you take another look?

Sighery committed:
Make connection login and password TEXT

This commit will change the connection `login` and `password` fields
to be of type `TEXT`, so it won't have a length limitation anymore.
